swan-lake/development-tutorials/additional-tools/scan-tool.md

```diff
@@ -85,6 +85,64 @@ To ignore a specific set of rules during the analysis, use the following command
 $ bal scan --exclude-rules="ballerina:1, ballerina/io:2"
 ```
 
-## Publishing static code analysis reports to SonarQube.
-
-To learn how to publish reports to SonarQube, refer to [Configuration for Platform Plugins](https://github.com/ballerina-platform/static-code-analysis-tool/blob/main/docs/static-code-analysis-tool/ScanFileConfigurations.md#configuration-for-platform-plugins).
\ No newline at end of file
+## Publish static code analysis reports to SonarQube.
+
+SonarQube is a popular open-source platform for continuous inspection of code quality. It provides static code analysis, code coverage, and other features to help developers maintain clean, maintainable codebases. The Ballerina scan tool can be integrated with SonarQube to publish static code analysis reports, enabling seamless integration into your CI/CD pipelines.
+
+This guide walks you through the process of configuring SonarQube and publishing Ballerina static code analysis reports.
+
+### Prerequisites
+- SonarQube 9.9 LTS installed.
+- SonarScanner CLI 4.8.0 or later installed and added to your system `PATH`.
+- [SonarQube Ballerina plugin](TODO), and [SonarQube platform plugin](TODO) downloaded.
+
+### Configure the SonarQube server
+
+1. Download and Install SonarQube 9.9 LTS.
```

Comment:
If we can give a link to the download, its better
wdyt?

Reply:
Fixed with 6d811265f7b9ad804bf3c3eb7e0ff960ed934843